h/static/scripts/annotator/guest.coffee

```diff
@@ -1,7 +1,20 @@
-Promise = require('es6-promise').Promise
+raf = require('raf')
+Promise = global.Promise ? require('es6-promise').Promise
```

Comment:
This pattern happens quite a bit. It's important because the es6-promise lib doesn't appear smart enough to delegate to the native implementation and using the polyfill has a negative performance impact due to setTimeout.